frontend/alert/components/AlertForm/index.tsx

```diff
@@ -184,10 +211,15 @@ const AlertForm = ({
         Promise.allSettled(promises)
             .then((responses) => responses.forEach(
                 (res: PromiseSettledResult<Response>, i) => {
+                    //fulfilled if response is returned, even if reg is unsuccessful.
```

Comment:
Turns out this behavior (accept on any response code) comes from `fetch` (used in `doAPIRequest`): https://stackoverflow.com/a/38236296
This makes sense, because looking at the code above `handleError` displays an error message saying:
> "We're sorry, but there was an error sending your request to our servers. Please try again!"

so it wouldn't make sense for `handleError` to be called in the case of non-2xx responses.

Reply:
Ah that make sense!